Models/DTOs/FreelancerDTO.cs

```diff
@@ -6,7 +6,8 @@ namespace AonFreelancing.Models.DTOs
    
     public class FreelancerResponseDTO : UserResponseDTO
     {
-        public List<SkillOutputDTO>Skills { get; set; }
```

Comment:
Why removed `Skills`.?

Reply:
there is already a dedicated endpoint for getting skills using pagination, so it is not needed, here.

Reply:
You mean the skills have a stand-alone endpoint to get the freelancer's skills, and it will be triggered at the profile web page?

Reply:
Yes